packages/server/test/support/fixtures/projects/session/cypress.json

```
@@ -0,0 +1,3 @@
+{
+  "experimentalSessionSupport": true
```

Unnecessarily generated files - can remove them from git.

```suggestion
  "experimentalSessionSupport": true,
  "supportFile": false,
  "fixturesFolder": false
```